dffml/noasync.py

```diff
@@ -114,3 +114,90 @@ def cleanup():
         "    >>> from dffml import *\n    >>> from dffml.noasync import *\n",
     )
 )
+
+
+def run(*args, **kwargs):
+    """
+    >>> # Socket server derived from
+    >>> # https://docs.python.org/3/library/socketserver.html#asynchronous-mixins
+    >>> import socket
+    >>> import threading
+    >>> import socketserver
+    >>>
+    >>> from dffml.noasync import run
+    >>> from dffml import DataFlow, Input, op
+    >>>
+    >>> class ThreadedTCPRequestHandler(socketserver.BaseRequestHandler):
+    ...     def handle(self):
+    ...         data = str(self.request.recv(1024), "ascii")
+    ...         cur_thread = threading.current_thread()
+    ...         response = bytes("{}: {}".format(cur_thread.name, data), "ascii")
```

Comment:
Let's remove the use of the `threading`, we don't use the threads name, in fact, you remove it below. So let's just not add it in the first place.

Reply:
yes is removed it from the below, because if we run this code on different machines the thread names will come differently, hence the tests will fail sometimes.